development/webpack.config.js

```diff
@@ -24,9 +24,14 @@ module.exports = {
 	},
 	devServer: {
 		static: {
-			directory: path.join(__dirname),
+			directory: path.join(__dirname, "public"),
 		},
-		watchFiles: ["../**/**"],
+		watchFiles: [
+			"./src",
+			"./*.{js,json,ts,html}",
+			"../src",
+			"../*.{js,json,ts,html}",
```

Comment:
Lets add `./.env` in here too so environment variable files are caught

Reply:
It doesn't seem that runtime changes to the values of the variables are supported by the `dotenv-webpack` plugin, currently. If I add `.env` to the watched files, the dev server will reload for changes to the `.env` file, but the values of any changed variables are not updated in the app until after restart.

See the current open issue here: https://github.com/mrsteele/dotenv-webpack/issues/476

Reply:
Ah! Fair enough - I looked at the original thread and looks like there's a solution but we can do that separately another time. This looks like it is an improvement regardless so let's get it in 🙂